test/e2e/config/digitalocean-ci.yaml

```diff
@@ -46,7 +46,7 @@ providers:
   - name: digitalocean
     type: InfrastructureProvider
     versions:
-    - name: v1.1.0
+    - name: v1.1.99
```

Comment:
Just to confirm, is `99` intended here?

Reply:
yep, this is just to we not forget and keeping update everytime